src/main/java/lesson3/task1/Seat.java

```diff
@@ -0,0 +1,44 @@
+package lesson3.task1;
+
+public class Seat {
+    private int seatNumber;
+    private int rowNumber;
+    private boolean available;
+
+    public Seat(int seatNumber, int rowNumber, boolean available) {
+        this.seatNumber = seatNumber;
+        this.rowNumber = rowNumber;
+        this.available = available;
+    }
+
+    public static String reserveSeat(Seat seatToReserve) {
```

Comment:
don't do this method as static. try to refactor it in such a way that reserveSeat will change "this.available" not incoming Seat object